manic/checkout.py

```diff
@@ -281,12 +283,21 @@ def main(args):
                 msg = str(tree_status[comp])
                 printlog(msg)
             # exit gracefully
-            msg = textwrap.fill(
-                'Some external repositories are not in a clean state. '
-                '(Generally, these are repositories with "M" in the '
-                'second column in the above status output.) '
-                'Please ensure all external repositories are clean '
-                'before updating.')
+            msg = """The external repositories labeled with 'M' above are not in a clean state.
+
+The following are two options for how to proceed:
+
+(1) Go into each external that is not in a clean state and issue either
+    an 'svn status' or a 'git status' command. Either revert or commit
+    your changes so that all externals are in a clean state. (Note,
+    though, that it is okay to have untracked files in your working
+    directory.) Then rerun {progname}.
+
+(2) Alternatively, you do not have to rely on {progname}. Instead, you
+    can manually update out-of-sync externals (labeled with 's' above)
```

Comment:
Is it worth replacing 'M' and 's' with {modified} and {out_of_sync}?

Reply:
Sorry, I'm not clear on what you're suggesting here; can you clarify?